src/initialize-custom-elements.ts

```diff
@@ -1,54 +1,12 @@
 import Application from '@glimmer/application';
+import glimmerElementFactory from './glimmer-element';
 
-export default function initializeCustomElements(app: Application, customElementDefinitions: string[]): void {
-  customElementDefinitions.forEach((name) => {
-    initializeCustomElement(app, name);
+export function initializeCustomElements(app: Application, customElementDefinitions: string[]): void {
+  customElementDefinitions.forEach(name => {
+    initializeCustomElement(app, name, name, []);
```

Comment:
Yes, probably. I was neglecting `initializeCustomElements` since I wasn't using it for my demo app. I think this whole file could use some more love.